jormungandr/src/explorer/graphql/config_param.rs

```diff
@@ -0,0 +1,525 @@
+use super::{scalars::Value, BftLeader, Ratio, TaxType};
+use async_graphql::{Enum, SimpleObject, Union};
+use chain_addr::Discrimination as DiscriminationLib;
+use chain_impl_mockchain::{
+    chaintypes::ConsensusType as ContainerTypeLib,
+    config::{
+        Block0Date as Block0DateLib, ConfigParam as ConfigParamLib, RewardParams as RewardParamsLib,
+    },
+    fee::{
+        LinearFee as LinearFeeLib, PerCertificateFee as PerCertificateFeeLib,
+        PerVoteCertificateFee as PerVoteCertificateFeeLib,
+    },
+    fragment::ConfigParams as ConfigParamsLib,
+    key::BftLeaderId,
+    milli::Milli as MilliLib,
+    rewards::{Ratio as RatioLib, TaxType as TaxTypeLib},
+    value::Value as ValueLib,
+    vote::CommitteeId,
+};
+use std::num::{NonZeroU32, NonZeroU64};
+
+#[derive(SimpleObject)]
+pub struct Block0Date {
+    block_0_date: u64,
```

Comment:
I think we usually spell this as `block0_date` for this. Not sure what's the most correct one, but it may be better to keep it consistent?